src/components/TasksTable/index.jsx

```diff
@@ -335,47 +413,57 @@ export default class TasksTable extends Component {
             </Button>
           </div>
         )}
-        <div className={classes.tableWrapper}>
-          <DataTable
-            items={data}
+        {displayCardLayout || this.props.width === 'xs' ? (
```

Comment:
You can probably use the `Hidden` component instead of `this.props.width === 'xs'`. https://material-ui.com/layout/hidden/#hidden